resources/main-template.cc

```diff
@@ -279,6 +312,30 @@ static PVOID old_vectored_exception_handler;
 
 namespace boxednode {
 
+#if HAVE_OPENSSL
```

Comment:
Everything below here is basically taken straight from Node.js itself; once the PR mentioned above is merged and released, these code paths don’t need to be taken anymore